core/src/main/java/com/netease/arctic/iceberg/CombinedDeleteFilter.java

```diff
@@ -416,4 +430,51 @@ public RecordWithLsn recordCopy() {
       return this;
     }
   }
+
+  private class CloseableIterableWithOtherCloseable<T> implements CloseableIterable<T> {
```

Comment:
could it be named as CloseableIterableWrapper?

Reply:
ok